arm-notificationhubs/2016-03-01/swagger/notificationhubs.json

```diff
@@ -0,0 +1,1742 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "NotificationHubsManagementClient",
+        "description": "Azure NotificationHub client",
+        "version": "2016-03-01"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json",
+        "text/json"
+    ],
+    "produces": [
+        "application/json",
+        "text/json"
+    ],
+    "security": [{
+        "azure_auth": [
+            "user_impersonation"
+        ]
+    }],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.NotificationHubs/checkNamespaceAvailability": {
+            "post": {
+                "tags": [
+                    "Namespaces"
+                ],
+                "operationId": "Namespaces_CheckAvailability",
+                "description": "Checks the availability of the given service namespace across all Windows Azure subscriptions. This is useful because the domain name is created based on the service namespace name.",
+                "externalDocs": {
+                    "url": "http://msdn.microsoft.com/en-us/library/windowsazure/jj870968.aspx"
+                },
+                "parameters": [{
+                    "name": "parameters",
+                    "in": "body",
+                    "required": true,
+                    "schema": {
+                        "$ref": "#/definitions/CheckAvailabilityParameters"
+                    },
+                    "description": "The namespace name."
+                }, {
+                    "$ref": "#/parameters/ApiVersionParameter"
+                }, {
+                    "$ref": "#/parameters/SubscriptionIdParameter"
+                }],
+                "responses": {
+                    "200": {
+                        "description": "",
```

Comment:
empty or missing description not allowed. Please change this everywhere in the spec.